website/learn/unique-semantics.md

```diff
@@ -323,7 +323,7 @@ representation.
 
 If any error happens during the first two phases, Elvish rejects the chunk
 without executing any of it. For instance, in Elvish unclosed parenthesis is a
-syntax error (error in the first parsing phase). The following code, when
+syntax error during the parsing phase. The following code, when
```

Comment:
This makes it sound like that there could be syntax errors during other phases, so just say "an error during the parsing phrase".